android/app/build.gradle

```diff
@@ -217,6 +217,7 @@ dependencies {
     implementation project(':react-native-custom-tabs')
     implementation project(':react-native-device-info')
     implementation project(':react-native-google-analytics-bridge')
+    compile project(':react-native-interactable')
```

Comment:
Change this to "implementation"

Reply:
Can you point me in the direction as to why? If I’m differing from the docs here

Reply:
Ah. Deprecated. I’ll change it.